src/main/resources/db/migration/V1__init.sql

```diff
@@ -16,4 +16,29 @@ create table annotations
     variant_id bigint not null references variants,
     info       jsonb  not null,
     db_name    varchar
-)
+);
+
+
+create table users (
+  id bigserial primary key,
+  username varchar not null,
+  password varchar not null
+);
+
+create table roles (
+  id bigserial primary key,
+  name varchar not null
+);
+
+create table users_roles (
+  user_id bigserial not null,
+  roles_id bigserial not null,
+  constraint user_fk foreign key (user_id) references users (id),
+  constraint role_fk foreign key (roles_id) references roles (id)
+);
+
+INSERT INTO roles (name) VALUES ('ROLE_USER');
+INSERT INTO roles (name) VALUES ('ROLE_ADMIN');
+
+INSERT INTO users (username, password) VALUES ('admin', '$2a$10$a7OsO0RhFaKziKginvn1.ON95coPosmBE0InUuZB1OLmDzps6t6lu');
```

Comment:
Do we need these two inserts?

Reply:
No, I meant that I doubt the correctness of the demonstration of these scripts on the githab

Reply:
I think it's better not to put these inserts on githab